PWGCF/FemtoWorld/TableProducer/femtoWorldProducerReducedTask.cxx

```diff
@@ -28,7 +28,7 @@
 #include "Common/DataModel/Multiplicity.h"
 #include "ReconstructionDataFormats/Track.h"
 #include "Common/Core/trackUtilities.h"
-#include "PWGLF/DataModel/LFStrangenessTables.h"
+#include "Common/DataModel/StrangenessTables.h"
```

Comment:
Should it remain in LF? What was the change discussed yesterday?

Reply:
It was moved to the PWGLF, should I create a copy for the PWGCF? Is that what you mean?

Reply:
No, I meant why you changed this include line to not-LF. But the checks passed, though the file is no longer in Common/DataModel ...

Reply:
Ah... It passed because for now I'm not using this task. It's commented in the CMakeList. I know it shouldn't be like that, in the future all of the tasks will be included in the CMakeList. But I'll change that to the `#include "PWGLF/DataModel/LFStrangenessTables.h"`